From: "Ying-Chun Liu (PaulLiu)" <paulliu@debian.org>
Date: Fri, 10 Apr 2026 05:38:47 +0000
Subject: Temporarily disable fapi-io tests

Forwarded: not-needed
Bug: https://github.com/tpm2-software/tpm2-tss/issues/2135

This unit test is not working. And it actually doesn't affect anything.
Disable it until upstream fixes this.
===================================================================
---
 test/unit/fapi-io.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/test/unit/fapi-io.c b/test/unit/fapi-io.c
index 8466376..c8bd103 100644
--- a/test/unit/fapi-io.c
+++ b/test/unit/fapi-io.c
@@ -372,10 +372,6 @@ main(int argc, char *argv[])
 #endif
 
     const struct CMUnitTest tests[] = {
-        cmocka_unit_test(check_io_read_async),
-        cmocka_unit_test(check_io_read_finish),
-        cmocka_unit_test(check_io_write_async),
-        cmocka_unit_test(check_io_write_finish),
     };
     return cmocka_run_group_tests(tests, NULL, NULL);
 }
